docs/user-guide/doc-odm-user-guide/studybrowser.md

```diff
@@ -1,84 +1,112 @@
+To do this:
 
-### Results panel
+1. Click the cog icon at the bottom of the filter panel to change the facets.
+2. Add facets by clicking the **New Facet** button.
+3. Type the exact name of the metadata field (names are case-sensitive).
 
-![image](doc-odm-user-guide/images/quickstart_user_studybrowser.png)
+You can reorder facets by dragging the icon next to the facet name or delete them by clicking the bin icon. Click the **Save** button to apply changes. The panel will not save if your facet is empty or a duplicate.
 
-The main panel in the study browser shows the results of your search, or if no search terms/filters have been applied, all studies that you have visibility of. The first column displays the name of the study, and you can click this to look at the study in more detail. It also lists information about which user created or imported the study, and the date.
+![13Facets.png](doc-odm-user-guide/images/13Facets.png)
+<figcaption>Configure facets. Reorder the facets to display for searching studies. Click on <strong>Configure facets</strong> at the bottom left of the page. A new window will open where you can add, rename, delete or drag facets. The changes will apply after clicking on <strong>Save</strong></figcaption>
 
-To the left of the study titles is a three dot link. Click this to open a menu that allows you to share the study (if you have permission), export data, copy the accession of the study, add it to your bookmarks, or view more information.
+!!! Important
+    You can only access the Configure facets option if you have the necessary permissions. For access, an admin must edit the settings. Refer to the [User Setup](setup.md) section for more information.
 
-![image](doc-odm-user-guide/images/three_dots_menu.png)
+## Navigation and Help
 
-Under the study title there is a summary of the metadata that is associated with the study. This displays information such as the organism, tissue, cell-type, disease and so on and is pulled straight from the metadata fields of the samples in the study.
+* Use the shortcut dock in the top left-hand corner of any window to return to the main dashboard.
+* Click on Quick Guide in the top right of your window for reference guides and examples on how to use ODM.
 
-You can hover over any name in the summary column and the name of the metadata field where the data comes from will appear.
+![14Navigation.png](doc-odm-user-guide/images/14Navigation.png)
+<figcaption>Navigation and help. Click on the left panel to return to the main dashboard. Click on the button Quick Guide on the right to explore Quick User Guides</figcaption>
```

Comment:
It seems to me that there's a mistake here, after clicking on the "Quick Guide" button youtube will be opened, not user guide